core/kubernetes/openhim/openhim-console-deployment.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: openhim-console-deployment
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      component: openhim-console
+  template:
+    metadata:
+      labels:
+        component: openhim-console
+    spec:
+      containers:
+        - name: openhim-console
+          image: jembi/openhim-console:1.13
+          ports:
+            - containerPort: 80
+          volumeMounts:
+            - name: console-config
+              mountPath: /usr/share/nginx/html/config/default.json
+              subPath: default.json
+          env:
+            - name: NODE_ENV
+              value: development
```

Comment:
I dont think we use these environment variables for the console so they could be removed